libzip subproject - cannot find size of the `off_t` type with Muon

### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Fedora 35 x86_64
| File format of the file you reverse (mandatory)      | -
| Architecture/bits of the file (mandatory)            | -
| `rizin -v` full output, **not truncated** (mandatory)  | 7c5e4f1fad87be2fbb7eff94a9f2dd2cba759410

### Expected behavior

No error

### Actual behavior

```
[1063/1649] compiling c subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add.c.o
FAILED: subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add.c.o 
cc -I subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3/lib -I subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3 -std=c99 -O3 -Wpedantic -Wextra -Wall -Werror=sizeof-pointer-memaccess -MD -MQ subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add.c.o -MF subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add.c.o.d -o subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add.c.o -c ../subprojects/libzip-1.7.3/lib/zip_add.c
In file included from ../subprojects/libzip-1.7.3/lib/zipint.h:38,
                 from ../subprojects/libzip-1.7.3/lib/zip_add.c:36:
../subprojects/libzip-1.7.3/lib/compat.h:146:2: error: #error unsupported size of off_t
  146 | #error unsupported size of off_t
      |  ^~~~~
[1064/1649] /home/xvilka/rizin/muon/build/muon internal exe -- /usr/bin/python3 /home/xvilka/rizin/rizin_copy/subprojects/libzip-1.7.3/create_zip_err_str.py /home/xvilka/rizin/rizin_copy/build/subprojects/libzip-1.7.3/zip_err_str.c /home/xvilka/rizin/rizin_copy/subprojects/libzip-1.7.3/lib/zip.h
[1065/1649] compiling c subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_entry.c.o
FAILED: subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_entry.c.o 
cc -I subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3/lib -I subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3 -std=c99 -O3 -Wpedantic -Wextra -Wall -Werror=sizeof-pointer-memaccess -MD -MQ subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_entry.c.o -MF subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_entry.c.o.d -o subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_entry.c.o -c ../subprojects/libzip-1.7.3/lib/zip_add_entry.c
In file included from ../subprojects/libzip-1.7.3/lib/zipint.h:38,
                 from ../subprojects/libzip-1.7.3/lib/zip_add_entry.c:37:
../subprojects/libzip-1.7.3/lib/compat.h:146:2: error: #error unsupported size of off_t
  146 | #error unsupported size of off_t
      |  ^~~~~
[1066/1649] compiling c subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_dir.c.o
FAILED: subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_dir.c.o 
cc -I subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3/lib -I subprojects/libzip-1.7.3 -I ../subprojects/libzip-1.7.3 -std=c99 -O3 -Wpedantic -Wextra -Wall -Werror=sizeof-pointer-memaccess -MD -MQ subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_dir.c.o -MF subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_dir.c.o.d -o subprojects/libzip-1.7.3/liblibzip.a.p/lib/zip_add_dir.c.o -c ../subprojects/libzip-1.7.3/lib/zip_add_dir.c
In file included from ../subprojects/libzip-1.7.3/lib/zipint.h:38,
                 from ../subprojects/libzip-1.7.3/lib/zip_add_dir.c:36:
../subprojects/libzip-1.7.3/lib/compat.h:146:2: error: #error unsupported size of off_t
  146 | #error unsupported size of off_t
      |  ^~~~~
```

### Steps to reproduce the behavior
1. `git clone https://git.sr.ht/~lattis/muon`
2. `cd muon; ./bootstrap build; build/muon setup build; ninja -C build`
3. `cd rizin; ../muon/build/muon setup build; ninja -C build`

### Additional Logs, screenshots, source code,  configuration dump, ...

Full building log:
[muon_setup.log.zip](https://github.com/rizinorg/rizin/files/8122030/muon_setup.log.zip)
[muon_ninja.log.zip](https://github.com/rizinorg/rizin/files/8122031/muon_ninja.log.zip)

--- a/subprojects/packagefiles/libzip-1.7.3/cmake-config.h.in
+++ b/subprojects/packagefiles/libzip-1.7.3/cmake-config.h.in
@@ -46,8 +46,8 @@
 #cmakedefine HAVE_STRINGS_H
 #cmakedefine HAVE_UNISTD_H
 #cmakedefine HAVE_WINDOWS_CRYPTO
-#cmakedefine SIZEOF_OFF_T @SIZEOF_OFF_T@
-#cmakedefine SIZEOF_SIZE_T @SIZEOF_SIZE_T@
+#cmakedefine SIZEOF_OFF_T
+#cmakedefine SIZEOF_SIZE_T
 #cmakedefine HAVE_DIRENT_H
 #cmakedefine HAVE_FTS_H
 #cmakedefine HAVE_NDIR_H

